hypothesis-python/tests/cover/test_conjecture_engine.py

```diff
@@ -603,13 +603,13 @@ def f(data):
 
 @pytest.mark.parametrize('count', [1, 3])
 def test_debug_data(capsys, count):
-    with settings(verbosity=Verbosity.debug):
-        @run_to_buffer
-        def f(data):
-            for _ in hrange(count):
-                data.draw_bytes(1)
-            if sum(data.buffer) > 10:
-                data.mark_interesting()
+    @settings(verbosity=Verbosity.debug)
+    @run_to_buffer
```

Comment:
IIRC this must be the outermost decorator, as it calls the wrapped function and returns the underlying buffer consumed by Conjecture.